### PR TITLE
feat: Expose more CORS related configurations via environment variables

### PR DESCRIPTION
**Pull request by SaschaPWittwer**
With the current django configurations exposed, it was not possible to deploy and use caluma in a CORS scenario. 
With this change more CORS header configurations are exposed via environment variables. Furthermore it enhances the django default values with some caluma required stuff like the language header.

### CODE REVIEW

**Comment by anehx** [comment]
Now the CI is failing because of the linter. Please run `docker-compose exec caluma black .`

File: caluma/settings/django.py [hunk 1]
```diff
@@ -130,6 +130,28 @@ def parse_admins(admins):  # pragma: no cover
...
+    'x-csrftoken',
+    'x-requested-with'
+])
+CORS_ALLOW_METHODS = env.list("CORS_ALLOW_METHODS", default=[
+    'DELETE',
+    'GET',
+    'OPTIONS',
+    'PATCH',
+    'POST',
+    'PUT'
```
[the remark this answers is not in the document]
Comment (open-dynaMIX) [reply caluma/settings/django.py]: Sorry to intervene. Actually we use GET requests for the `/healthz` endpoint.

[the remark this answers is not in the document]
Comment (anehx) [reply caluma/settings/django.py]: Ah right I forgot about that! Sorry @SaschaPWittwer !

[the remark this answers is not in the document]
Comment (SaschaPWittwer) [reply caluma/settings/django.py]: Ahhh good one! I will update the default value once again.

**Review by anehx — APPROVED** [review]

**Comment by anehx** [comment]
@open-dynaMIX could you review this once again?

**Review by open-dynaMIX — APPROVED** [review]
Awesome, thanks a lot! :tada:

**Review by open-dynaMIX — CHANGES_REQUESTED** [review]
Sorry, forgot to check before. @SaschaPWittwer could you please merge all thos commits into one in order to keep the history clean? Thanks a lot!

**Comment by anehx** [comment]
@open-dynaMIX let's just squash..

**Pull request merged**